falcon/hooks.py

```diff
@@ -27,7 +27,7 @@
     '|'.join(method.lower() for method in COMBINED_METHODS)))
 
 
-def before(action, *args, **kwargs):
+def before(action, *args, is_async=None, **kwargs):
```

Comment:
I could also add a `falcon.asgi.hooks()` that just always sets this argument to `True`, but I'm not certain how helpful that would be. Thoughts?